src/messages/MMgrReport.h

```diff
@@ -30,6 +31,7 @@ class PerfCounterType
   std::string nick;
   enum perfcounter_type_d type;
 
+
```

Comment:
Nit: drop blank line

Reply:
Fixed in: https://github.com/ceph/ceph/pull/24215/commits/3a5f9989c9b6402152b475ba295d4b88c2470906